src/main/resources/static/cas1-schemas.yml

```diff
@@ -472,15 +472,14 @@ components:
         - canonicalArrivalDate
         - canonicalDepartureDate
         - releaseType
-        - spaceType
       x-enum-varnames:
         - PERSON_NAME
         - TIER
         - CANONICAL_ARRIVAL_DATE
         - CANONICAL_DEPARTURE_DATE
         - RELEASE_TYPE
         - SPACE_TYPE
-    Cas1PremiseDaySummary:
+    Cas1PremisesDaySummary:
```

Comment:
sorry, this is my fault. Fred taught me this week that the singular of Premises is Premises :)  Thanks for fixing it